shell/docker-build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Ensure we fail the job if any steps fail
+# Do not set -u as DOCKER_ARGS may be unbound
+set -e -o pipefail
+
+# Switch to the directory where the Dockerfile is
+cd "$DOCKER_ROOT"
+
+# Build the docker image
+
+# Allow word splitting
+# shellcheck disable=SC2086
+docker build $DOCKER_ARGS . | tee "$WORKSPACE/docker_build_log.txt"
+DOCKER_IMAGE=$(grep -Po 'Successfully built \K[^ ]+' \
+    "$WORKSPACE/docker_build_log.txt")
+
+IMAGE_NAME="$DOCKERREGISTRY/$DOCKER_NAME:$DOCKER_TAG"
```

Comment:
Actually, the comment for it is below this line. I didn't originally have the variable being set as I only needed the combined value once.